integration-tests/data/snaps/basic-binaries/meta/snap.yaml

```diff
@@ -0,0 +1,17 @@
+name: basic-binaries
+version: 1.0
+icon: meta/snappy64.png
+apps:
+ echo:
+  command: bin/echo
+  uses: [nothing]
+ success:
+  command: bin/success
+  uses: [nothing]
+ fail:
+  command: bin/fail
+  uses: [nothing]
+uses:
```

Comment:
The default permissions for apps are `caps: [network-client]`. This is giving the basic binaries less permissions than this. But yeah, we don't really need this, we could use the default permissions. But note that we currently do not add the migration-skill by default. Maybe we should during the transition period.